src/Management/src/EndpointBase/HeapDump/HeapDumper.cs

```diff
@@ -50,7 +66,30 @@ public string DumpHeap()
 
         internal string CreateFileName()
         {
-            return "minidump-" + DateTime.Now.ToString("yyyy-MM-dd-HH-mm-ss") + "-live" + ".dmp";
+            if (Environment.Version.Major == 3 || "gcdump".Equals(_options.HeapDumpType, StringComparison.OrdinalIgnoreCase))
+            {
+                return "gcdump-" + DateTime.Now.ToString("yyyy-MM-dd-HH-mm-ss") + "-live" + ".gcdump";
```

Comment:
Has this scenario been tested on a recent version of Apps Manager? I'm not 100% sure what the current state of file extension handling is here

Reply:
Tested, and as expected the Apps Manager renames the file incorrectly and VS2022 does not open the file correctly. If the file is renamed to include .gcdump extension, then all works as expected